scripts/issue/595/migrate_tcl.py

```diff
@@ -0,0 +1,26 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+
+def main():
+    vault = GreatApeSafe(r.badger_wallets.treasury_vault_multisig)
+    vault.init_balancer()
+    vault.init_aura()
+    
+
+    bpt = vault.contract(r.balancer.B_20_BTC_80_BADGER)
+    # mainly to ensure no dust is left on snap
+    vault.take_snapshot(tokens=[bpt.address])
+
+    (_, _, _, rewards) = vault.aura.get_pool_info(bpt)
+    rewards = vault.contract(rewards)
+
+    vault.balancer.unstake_all(bpt)
+
+    bpt_amount = bpt.balanceOf(vault)
+
+    bal_before_stake = rewards.balanceOf(vault)
+    vault.aura.deposit_and_stake(bpt, bpt_amount)
```

Comment:
I can't seem to find the code for this function, can you please link?
Aura seems like it doesn't have this function:
https://github.com/Badger-Finance/badger-multisig/blob/2cb800d5babb226a1cde1d3a9129fcbd91864aa1/great_ape_safe/ape_api/aura.py